NeosModLoader/NeosModLoader.csproj

```diff
@@ -40,18 +40,27 @@
     <Reference Include="FrooxEngine">
       <HintPath>C:\Program Files (x86)\Steam\steamapps\common\NeosVR\Neos_Data\Managed\FrooxEngine.dll</HintPath>
     </Reference>
+    <Reference Include="Newtonsoft.Json">
+      <HintPath>C:\Program Files (x86)\Steam\steamapps\common\NeosVR\Neos_Data\Managed\Newtonsoft.Json.dll</HintPath>

```

Comment:
Would this hardcoded location be an issue for people who have their Neos install somewhere else? I'm not 100% sure how `<HintPath>` works with external libraries.

Reply:
nope, this is just the path used for compiling.